src/api_midlevel.jl

```diff
@@ -185,3 +185,17 @@ function write_chunk(dataset_id, index::Integer, buf::Union{DenseArray,Base.Fast
     offset = [reverse(get_chunk_offset(dataset_id, index))...]
     write_chunk(dataset_id, offset, buf; dxpl_id = dxpl_id, filter_mask = filter_mask)
 end
+
+function get_fill_value(plist_id, ::Type{T}) where T

```

Comment:
Would the second argument ever not be `Float64`?

Reply:
One might want an `Int64` at some point.

Reply:
I defaulted to `Float64` since that should cover the most common cases.

Reply:
Ok I see that that is the case in the test examples. And it doesn't seem like something we could infer.

Reply:
Right. I cannot find a way to find the type unless we have a reference to the associated dataset.